pkg/drivers/kic/oci/network_create.go

```diff
@@ -38,14 +38,30 @@ const firstSubnetAddr = "192.168.49.0"
 const defaultSubnetMask = 24
 
 // name of the default bridge network, used to lookup the MTU (see #9528)
-const defaultBridgeName = "bridge"
+const dockerDefaultBridge = "bridge"
+
+// name of the default bridge network
+const podmanDefaultBridge = "podman"
 
 // CreateNetwork creates a network returns gateway and error, minikube creates one network per cluster
 func CreateNetwork(ociBin string, name string) (net.IP, error) {
 	return createDockerNetwork(ociBin, name)
 }
 
 func createDockerNetwork(ociBin string, clusterName string) (net.IP, error) {
+	var defaultBridgeName string
+	if ociBin == Docker {
+		defaultBridgeName = dockerDefaultBridge
+	}
+	if ociBin == Podman {
+		defaultBridgeName = podmanDefaultBridge
+	}
+
+	// validate the name of the new network
+	if clusterName == defaultBridgeName {
+		return nil, errors.Errorf("Can't use same name as %s's default network: %s", ociBin, clusterName)
```

Comment:
I wonder if we should not Error, but out.Warnning
that you specified the DefaultNetwork, we will Not create a new one and just use that.
I think this will come handy when a paranoid user like me wants to ensure the network is bridge and not create a minikube network. 

some users need to Use the default bridge maybe for weird MTU issues we seen in the cloud shell

Reply:
I recommend checking before creating if the name is Bridge then just return the net.IP of that bridge.

Reply:
Just wondering what person names their cluster/profile to "bridge"